src/test/resources/projects/annotationsApply/build.gradle

```diff
@@ -0,0 +1,9 @@
+plugins {
+	id 'fabric-loom'
+}
+
+dependencies {
+	minecraft "com.mojang:minecraft:25w42a"
+	mappings "net.fabricmc:yarn:25w42a+build.9:v2"
+	modImplementation "net.fabricmc:fabric-loader:0.17.3"
+}
```

Comment:
For a test as simple as this just use the `minimalBase` project, and have a look at how the other tests use it, you can just move this dependencies block into the groovy test code. It doesnt look like you have a reason to have actual mod code for this test.

Reply:
Uhh, that shouldnt be the case (I wish it was) 🤔 Happy to go with this its not a big deal.